src/ui/components/componenttypes.js

```diff
@@ -0,0 +1,18 @@
+/** @module */
+
+/**
+ * The component types is a map that contains the types for all components
+ * given some unique key.
```

Comment:
We've effectively created an enum here, so I think it's okay to use that terminology. "An enum listing the different Component types supported in the SDK"

Reply:
For context, I was copying the registry jsdoc. I'll update